clarity_ext/dilution.py

```diff
@@ -220,9 +230,9 @@ def calculate_transfer_volumes(self):
                     transfer.buffer_volume *= scale_factor
                     transfer.scaled_up = True
             except (TypeError, ZeroDivisionError) as e:
-                transfer.sample_volume = None
-                transfer.buffer_volume = None
-                transfer.has_to_evaporate = None
+                transfer.sample_volume = 0
```

Comment:
Ok, I remember now how my reasoning did go by this. I would like an error and warnings detection code to be run within the snpseq script. Some of these checks have to be run after the transfer volumes have been calculated. Exampels of checks: Is the volume too high? Do the sample have to be evaporated? Does all samples have a source concentration. The try-block in this method is to avoid the program to crash at the first faulty sample, so that instead a list av all faulty samples can be shown to the user.

The error and warnings checks is not implemented yet. The validation method in this module is eventually going to be removed.